pkgs/development/libraries/tepl/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, autoconf, autoconf-archive, automake, pkgconfig, intltool
+, gobjectIntrospection, gtksourceview4, libuchardet, gnome3, gnome2
+, libtool, libxml2, amtk }:
+let
+  version = "4.1.1";
+in stdenv.mkDerivation {
+  name = "tepl-${version}";
+
+  src = fetchTarball {
+    url = "https://gitlab.gnome.org/GNOME/tepl/-/archive/${version}/tepl-${version}.tar.bz2";
+    sha256 = "01g6b5lryz7c40iv9j8x7kc0lzjq1r1mpzz5ljcqfmmy0pa4i91x";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    autoconf-archive
+    automake
+    pkgconfig
+    intltool
+    libtool
+    gobjectIntrospection
+    gnome2.gtkdoc
+  ];
+
+  buildInputs = [
+    libuchardet
+    gtksourceview4
+    libxml2
+    gnome3.gtk.dev
+    amtk
+  ];
+
+  configurePhase = ''
+    mkdir -p $out
+    ./autogen.sh --prefix=$out --disable-Werror
+  '';
+
```

Comment:
Could you also add updateScripts as per https://github.com/NixOS/nixpkgs/issues/36150